homeassistant/components/history/__init__.py

```diff
@@ -1,36 +1,28 @@
 """Provide pre-made queries on top of the recorder component."""
 from __future__ import annotations
 
-from collections import defaultdict
 from collections.abc import Iterable
 from datetime import datetime as dt, timedelta
-from itertools import groupby
-import json
 import logging
 import time
 from typing import cast
 
 from aiohttp import web
-from sqlalchemy import and_, bindparam, func, not_, or_
-from sqlalchemy.ext import baked
+from sqlalchemy import not_, or_
 import voluptuous as vol
 
-from homeassistant.components import recorder
 from homeassistant.components.http import HomeAssistantView
-from homeassistant.components.recorder.models import (
-    States,
-    process_timestamp,
-    process_timestamp_to_utc_isoformat,
-)
-from homeassistant.components.recorder.util import execute, session_scope
+from homeassistant.components.recorder.history import _get_significant_states
```

Comment:
Likely we need to import all the moved functions back here since custom components may rely on them as well, or make wrappers that warn the code is moved and using it from here is deprecated

Reply:
A wrapper is better, I'll add that.
Btw, do you think the split between what's in `recorder/history.py` and what's in `history/__init__.py` is OK?

Reply:
It seems reasonable to me.  I'd probably move `LazyState` into its own module maybe

existing -> `recorder/history/__init__.py`
LazyState -> `recorder/history/models.py`